scripts/shell/volreq.sh

```diff
@@ -345,6 +440,30 @@ function createDstDir() {
         echo -ne "$color_reset"
         exit 2
     fi
+
+    if [ "$umount_old_device" = "yes" ]; then
+      echo -ne "$color_red"
```

Comment:
aqui é um erro pra escrever em vermelho?

Reply:
Sure not! Fixed ;D